src/workers/email-report.js

```diff
@@ -0,0 +1,183 @@
+import { r } from '../server/models'
+import { sendEmail } from '../server/mail'
+import moment from 'moment'
+import { groupBy } from 'lodash'
+
+const SQL = (strings, ...values) => {
+  // Interweave the strings with the
+  // substitution vars first.
+  let output = ''
+  for (let i = 0; i < values.length; i += 1) {
+    output += strings[i] + values[i]
+  }
+  output += strings[values.length]
+
+  // Split on newlines.
+  const lines = output.split(/(?:\r\n|\n|\r)/)
+
+  // Rip out the leading whitespace.
+  return lines.map(line => line.replace(/^\s+/gm, '')).join(' ').trim()
+}
+
+(async () => {
```

Comment:
please make this file a library, and then enable running it with a separate file.
Then please add a method in workers/job-processes.js that runs it (see e.g. the run migrations function there) which can make it automatable the same way the other job processes are.